backend/products/dao/insert.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.append('..')
+from database.connection import *
+
+
+
+def product_insert(data):
+    '''
```

Comment:
See if you can refractor it and keep it modular.

Reply:
Okay